src/js/letters/utils/constants.js

```diff
@@ -1,34 +1,40 @@
 // Action types
+// getLetterList() actions
 export const BACKEND_AUTHENTICATION_ERROR = 'BACKEND_AUTHENTICATION_ERROR'; // 403
-export const INVALID_ADDRESS_PROPERTY = 'INVALID_ADDRESS_PROPERTY'; // 422
-export const LETTER_ELIGIBILITY_ERROR = 'LETTER_ELIGIBILITY_ERROR';  // 502
 export const BACKEND_SERVICE_ERROR = 'BACKEND_SERVICE_ERROR'; // 503 or 504
-
 export const GET_LETTERS_FAILURE = 'GET_LETTERS_FAILURE';
 export const GET_LETTERS_SUCCESS = 'GET_LETTERS_SUCCESS';
+export const INVALID_ADDRESS_PROPERTY = 'INVALID_ADDRESS_PROPERTY'; // 422
+export const LETTER_ELIGIBILITY_ERROR = 'LETTER_ELIGIBILITY_ERROR';  // 502
 
+// getMailingAddress() actions
 export const GET_ADDRESS_FAILURE = 'GET_ADDRESS_FAILURE';
 export const GET_ADDRESS_SUCCESS = 'GET_ADDRESS_SUCCESS';
 
-export const GET_BENEFIT_SUMMARY_OPTIONS_FAILURE = 'GET_BENEFIT_SUMMARY_OPTIONS_FAILURE';
-export const GET_BENEFIT_SUMMARY_OPTIONS_SUCCESS = 'GET_BENEFIT_SUMMARY_OPTIONS_SUCCESS';
-export const GET_LETTER_PDF_DOWNLOADING = 'GET_LETTER_PDF_DOWNLOADING';
-
-export const GET_LETTER_PDF_FAILURE = 'GET_LETTER_PDF_FAILURE';
-export const GET_LETTER_PDF_SUCCESS = 'GET_LETTER_PDF_SUCCESS';
+// getAddressCountries() actions
+export const GET_ADDRESS_COUNTRIES_FAILURE = 'GET_ADDRESS_COUNTRIES_FAILURE';
+export const GET_ADDRESS_COUNTRIES_SUCCESS = 'GET_ADDRESS_COUNTRIES_SUCCESS';
 
-export const UPDATE_BENFIT_SUMMARY_REQUEST_OPTION = 'UPDATE_BENFIT_SUMMARY_REQUEST_OPTION';
+// getAddressStates() actions
+export const GET_ADDRESS_STATES_FAILURE = 'GET_ADDRESS_STATES_FAILURE';
+export const GET_ADDRESS_STATES_SUCCESS = 'GET_ADDRESS_STATES_SUCCESS';
 
-export const UPDATE_ADDRESS = 'UPDATE_ADDRESS';
```

Comment:
There were no actions created and this was the only one removed...am I reading that right?

Reply:
yep!